### DISCUSSION TITLE
General Updates

### DISCUSSION THREAD

**Discussion by RamsesCaesar** [RamsesCaesar/CatAmongBirds#51] [Announcements]
This thread will be used for updates that aren't linked to any specific issue.

**Comment by RamsesCaesar** [comment]
I just added a new decoration to the main menu. (2b5ca305cfb542a9e7726116fe75b98962e4c942)


<img width="996" alt="Screenshot 2022-12-16 at 19 42 03" src="https://user-images.githubusercontent.com/66065017/208167515-39c2470f-9565-422e-bf74-4912f195379b.png">

**Comment by RamsesCaesar** [comment]
This is how the MainMenu looks now:

<img width="1436" alt="Screenshot 2022-12-17 at 17 38 17" src="https://user-images.githubusercontent.com/66065017/208252210-edcc7b4a-d299-4ad3-ab45-bd270b3debc0.png">

The  characters and background are animated in a continuous loop.

**Comment by RamsesCaesar** [comment]
After a winter pause, the game is now back in active development.

**Comment by TonyNashe** [comment]
Yeahhh
Am 28. Feb. 2023, 15:41 +0100 schrieb RamsesCaesar ***@***.***>:
